packages/components/src/slot-fill/bubbles-virtually/slot-fill-context.js

```diff
@@ -22,6 +22,7 @@ const SlotFillContext = createContext( {
 	unregisterSlot: () => {},
 	registerFill: () => {},
 	unregisterFill: () => {},
+	isDefault: true,
```

Comment:
Could we add a comment specifying what this isDefault is for? It seems not really a default more like a flag to understand if a slot exists in the hierarchy. Could it be named something like hasAncestorProvider?

Reply:
If we name it `hasAncestorProvider` it means we have to change the value to false and set the value to "true" when we actually use the provider. I prefer to avoid the latter and just mark the "default value of the context" as isDefault.

Reply:
I'd echo Jorge about that. It's hard to connect what `default` is.

Reply:
I'm going to add a comment here to clarify.